include/odp/api/spec/packet.h

```diff
@@ -1491,6 +1491,31 @@ void *odp_packet_user_area(odp_packet_t pkt);
  */
```

Comment:
I think this API is not necessary. User can simply set a pattern in the user area to check whether it is valid or not.

A single bit flag does not yield much benefit to the user in my opinion.

Reply:
When an application receives packets (from the scheduler), some are new packets from packet input HW and some are packet enqueued by the application. Application has set user area in those packets it has already processed, but those from packet input have random user area content.

Application can use this flag to check if user area content is random, or initialized properly.

Instead of the flag, we could specify e.g. that implementation always zeros the entire user area (or its first X bytes) on packet input/alloc/reset. But that would require implementation to write that specific memory area. This flag can locate anywhere in the packet header, and may be cleared automatically by HW (with any other metadata fields).

Reply:
Ack